mplex-yamux/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "libp2p-mplex-yamux"
```

Comment:
The library should be just `libp2p-yamux` (and directory named `yamux`).
`mplex` is the name of a protocol, not a shortcut for `multiplex`.